huaweicloud/resource_huaweicloud_cce_addon_v3_test.go

```diff
@@ -0,0 +1,137 @@
+package huaweicloud
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+
+	"github.com/huaweicloud/golangsdk/openstack/cce/v3/addons"
+)
+
+func TestAccCCEAddonV3_basic(t *testing.T) {
+	var addon addons.Addon
+
+	rName := fmt.Sprintf("tf-acc-test-%s", acctest.RandString(5))
+	resourceName := "huaweicloud_cce_addon_v3.test"
+	clusterName := "huaweicloud_cce_cluster_v3.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckCCEAddonV3Destroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCCEAddonV3_basic(rName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckCCEAddonV3Exists(resourceName, clusterName, &addon),
+					//resource.TestCheckResourceAttr(resourceName, "name", rName),
```

Comment:
should not leave comment line here, remove it or remove `//`

Reply:
fixed